server/auth/local/index.js

```diff
@@ -15,9 +18,19 @@ router.post('/', function(req, res, next) {
     if (!user) {
       return res.status(404).json({message: 'Something went wrong, please try again.'});
     }
+
     var profile = user.profile;
-    var token = signToken(user._id, user.role);
-    res.json({ token, profile });
+    User.findById(profile._id).then((user) => {
```

Comment:
These PRs have an order to them, I suppose this is a bad habit when under time constraint- this portion has already been merged